SU-SOE/soe_profile
# D8-CORE-2617: Added an icon for stories paragraphs

# READY FOR REVIEW

# Summary
- Adds an icon for the stories paragraph type.
- Matches this PR in the soe_paragraphs repo: https://github.com/SU-SOE/soe_paragraphs/pull/21/
- includes changes to default content to remove PHP serialized strings.

# Needed By (Date)
- today

# Urgency
- 4 of 10

# Steps to Test

1. pull this branch
2. `drush cim`
3. create a page, and notice there is a default icon for the stories paragraph type.

# Affected Projects or Products
- Does this PR impact any particular projects, products, or modules?

# Associated Issues and/or People
- JIRA ticket
- Other PRs
- Any other contextual information that might be helpful (e.g., description of a bug that this PR fixes, new functionality that it adds, etc.)
- Anyone who should be notified? (`@mention` them here)

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
